packages/kbn-esql-validation-autocomplete/src/validation/validation.ts

```diff
@@ -815,11 +814,6 @@ function validateSource(
     return messages;
   }
 
-  const hasCCS = hasCCSSource(source.name);
```

Comment:
ℹ️ Here we were bypassing the validation for CCS. I think is a leftover from when we were not supporting CCS. It is not needed anymore